hydra_base/lib/scenario.py

```diff
@@ -311,6 +313,59 @@ def purge_scenario(scenario_id, **kwargs):
     db.DBSession.flush()
     return 'OK'
 
+def create_child_scenario(scenario_id, child_name, **kwargs):
```

Comment:
Maybe call `scenario_id` -> `parent_scenario_id` to be more explicit?

Reply:
026aa448918f3857932bd386ead95eb43441499d